### ISSUE TITLE
Enable/Disable Peer

### ISSUE THREAD

**Issue by chnagy** [perara/wg-manager#108]
Great work so far.

I would be interested to enable and disable existing client entries. e.g. a checkbox which removes/adds the entry in the config.
So I can easily decide if and when they are able to connect to my server.

**Comment by perara** [comment]
Good Idea! This is definitely something that we should aim to provide. I'll add it to the "enhancement category" :)
